conditions/KubePodNotReady.json

```diff
@@ -1,19 +1,17 @@
 {
-  "conditions": [
-    {
-      "type": "alert_is_firing",
-      "alert": {
-        "name": "KubePodNotReady"
+    "conditions": [
+      {
+        "type": "alert_is_firing",
+        "alert": {
+          "name": "KubePodNotReady"
+        }
+      }
+    ],
+    "gathering_functions": {
+      "containers_logs": {
+        "alert_name": "KubePodNotReady",
+        "tail_lines": 100,
+        "previous": false
       }
-    }
-  ],
-  "gathering_functions": {
-    "containers_logs": {
-      "alert_name": "KubePodNotReady",
-      "tail_lines": 100
-    },
-    "pod_definition": {
-      "alert_name": "KubePodNotReady"
```

Comment:
I think we don't want to remove the `pod_definition` gathering.

Reply:
🦀 It seems that it was removed by the formatting tool. Weird. I'll fix it.